test/ConstExtraction/ExtractFromMacroExpansion.swift

```diff
@@ -7,6 +7,10 @@
 // RUN: %target-swift-frontend -typecheck -emit-const-values-path %t/ExtractFromMacroExpansion.swiftconstvalues -const-gather-protocols-file %t/protocols.json -primary-file %s -load-plugin-library %t/%target-library-name(MacroDefinition)
 // RUN: cat %t/ExtractFromMacroExpansion.swiftconstvalues 2>&1 | %FileCheck %s
 
+// Do the same, but ensure the WMO compilation flow produces the same result
+// RUN: %target-swift-frontend -typecheck -emit-const-values-path %t/ExtractFromMacroExpansionWMO.swiftconstvalues -const-gather-protocols-file %t/protocols.json -O %s -load-plugin-library %t/%target-library-name(MacroDefinition)
```

Comment:
Do we know for sure that -O will always enable WMO?

Reply:
It's not actually `-O` here, rather it is the absence of `-primary-file` directives, which triggers WMO and the new code-path.